scripts/cnv_wdl/germline/cnv_germline_case_scattered_workflow.wdl

```diff
@@ -0,0 +1,231 @@
+                gcnv_sample_psi_scale = gcnv_sample_psi_scale,
+                gcnv_depth_correction_tau = gcnv_depth_correction_tau,
+                gcnv_copy_number_posterior_expectation_mode = gcnv_copy_number_posterior_expectation_mode,
+                gcnv_active_class_padding_hybrid_mode = gcnv_active_class_padding_hybrid_mode,
+                gcnv_learning_rate = gcnv_learning_rate,
+                gcnv_adamax_beta_1 = gcnv_adamax_beta_1,
+                gcnv_adamax_beta_2 = gcnv_adamax_beta_2,
+                gcnv_log_emission_samples_per_round = gcnv_log_emission_samples_per_round,
+                gcnv_log_emission_sampling_median_rel_error = gcnv_log_emission_sampling_median_rel_error,
+                gcnv_log_emission_sampling_rounds = gcnv_log_emission_sampling_rounds,
+                gcnv_max_advi_iter_first_epoch = gcnv_max_advi_iter_first_epoch,
+                gcnv_max_advi_iter_subsequent_epochs = gcnv_max_advi_iter_subsequent_epochs,
+                gcnv_min_training_epochs = gcnv_min_training_epochs,
+                gcnv_max_training_epochs = gcnv_max_training_epochs,
+                gcnv_initial_temperature = gcnv_initial_temperature,
+                gcnv_num_thermal_advi_iters = gcnv_num_thermal_advi_iters,
+                gcnv_convergence_snr_averaging_window = gcnv_convergence_snr_averaging_window,
+                gcnv_convergence_snr_trigger_threshold = gcnv_convergence_snr_trigger_threshold,
+                gcnv_convergence_snr_countdown_window = gcnv_convergence_snr_countdown_window,
+                gcnv_max_calling_iters = gcnv_max_calling_iters,
+                gcnv_caller_update_convergence_threshold = gcnv_caller_update_convergence_threshold,
+                gcnv_caller_internal_admixing_rate = gcnv_caller_internal_admixing_rate,
+                gcnv_caller_external_admixing_rate = gcnv_caller_external_admixing_rate,
+                gcnv_disable_annealing = gcnv_disable_annealing,
+                ref_copy_number_autosomal_contigs = ref_copy_number_autosomal_contigs,
+                allosomal_contigs = allosomal_contigs 
+        }
+    }
+
+    output {
+        Array[File] contig_ploidy_calls_tars = CNVGermlineCaseWorkflow.contig_ploidy_calls_tar
+        Array[Array[Array[File]]] gcnv_calls_tars = CNVGermlineCaseWorkflow.gcnv_calls_tars
+        Array[Array[File]] gcnv_tracking_tars = CNVGermlineCaseWorkflow.gcnv_tracking_tars
+        Array[Array[File]] genotyped_intervals_vcf = CNVGermlineCaseWorkflow.genotyped_intervals_vcf
+        Array[Array[File]] genotyped_segments_vcf = CNVGermlineCaseWorkflow.genotyped_segments_vcf
+    }
+}
+
+task SplitInputArray {
+    Array[String] input_array
+    Int num_inputs_in_scatter_block
+    String gatk_docker
+
+    Int machine_mem_mb = 4000
+    Int disk_space_gb = 20
+    Int cpu = 1
+    Int preemptible_attempts = 5
+    Boolean use_ssd = false
+
+    File input_array_file = write_lines(input_array)
+
+    command <<<
+        python <<CODE
```

Comment:
perhaps a comment here describing what the python code is doing would be helpful.

Reply:
sounds good, done